src/storage-client/src/source/persist_source.rs

```diff
@@ -110,7 +107,8 @@ pub fn persist_source_core<G, YFn>(
     flow_control: Option<FlowControl<G>>,
     yield_fn: YFn,
 ) -> (
-    Stream<G, (Result<Row, DataflowError>, Timestamp, Diff)>,
+    Stream<G, (Row, Timestamp, Diff)>,
```

Comment:
why did this have to change?

Reply:
I should split this change out. The reasoning here is that `Row` is 32 bytes, but `DataflowError` is 80 bytes. Multiplexing them adds another 8 bytes to each element, which means we're wasting 56 bytes most of the time.

Reply:
👍 wfm!